server/src/main/java/com/objectcomputing/checkins/services/feedback_template/FeedbackTemplate.java

```diff
@@ -28,13 +28,11 @@ public class FeedbackTemplate {
 
     @Column(name = "title")
     @NotBlank
-    @TypeDef(type = DataType.STRING)
```

Comment:
Most of these TypeDef annotations were unnecessary...removed just for the sake of cleanliness.